### PR TITLE
add support for orchestrate to run sc tunnel, web server, and tests

### PR DESCRIPTION
**Pull request by mikedon**

### CODE REVIEW

File: test/e2e/configs/wdio.saucelabs-orchestrate.conf.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    /**
+     * Desktop browsers
+     */
+    {
+        browserName: 'chrome',
+        platformName: 'Windows 10',
+        browserVersion: 'latest',
+        'sauce:options': {
+            ...defaultBrowserSauceOptions,
+            tunnelIdentifier: "mdonovan2010_tunnel_name"
```
Comment (wswebcreation) [inline test/e2e/configs/wdio.saucelabs-orchestrate.conf.js:32]: Do you need to spin up the tunnel separately? WebdriverIO can do that for you, see [this](https://webdriver.io/docs/sauce-service#sauceconnect)

Comment (mikedon) [reply test/e2e/configs/wdio.saucelabs-orchestrate.conf.js:32]: when I did that I got an error related to spawning the separate process within node. It must be something related to doing it within orchestrate. This approach works though.

Comment (wswebcreation) [reply test/e2e/configs/wdio.saucelabs-orchestrate.conf.js:32]: ok, then I'm fine with it

**Review by wswebcreation — APPROVED** [review]
LGTM

**Pull request merged**